### PR TITLE
New package: Nanoleaf.NanoleafDesktop version 1.8.3

### LINKED ISSUE #163633: [Package Request]: Nanoleaf Desktop App
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://nanoleaf.me/en-US/integration/desktop-app/
Publisher: Nanoleaf
Package Name: Nanoleaf Desktop
Description: With the Nanoleaf Desktop App, you can control your 4D, Skylight, Lines, Shapes, Essentials*, Elements, Canvas or Light Panels from your computer (Windows or Mac). Create and download new Scenes, set Schedules and use your other favorite features right from your computer. The Desktop App was designed to simplify the management of larger installations and offer control of many different layouts at once.
Package Version: 1.8.3
Installer URL: https://my.nanoleaf.me/api/v2/desktopApp/latest/win

### PR DESCRIPTION
**Pull request by znarfm**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

Resolves: #163633

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/163794)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-07-19 06:04:54.528 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!6803535B: (caller: 67F31A80) LogHr(1) tid(1c70) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!6807B5DD: (caller: 681112BF) Exception(1) tid(1c70) 8A15005E ] 
> 2024-07-19 06:05:01.578 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!5454535B: (caller: 54441A80) LogHr(1) tid(18e4) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!5458B5DD: (caller: 546212BF) Exception(1) tid(18e4) 8A15005E ] 
> MSI (s) (F4:38) [06:05:07:696]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (F4:38) [06:05:07:696]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (F4:38) [06:05:07:696]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (F4:38) [06:05:07:696]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (F4:38) [06:05:07:696]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.
> Property(S): OutOfDiskSpace = 0
> Property(S): OutOfNoRbDiskSpace = 0
> Property(S): PrimaryVolumeSpaceAvailable = 0
> Property(S): PrimaryVolumeSpaceRequired = 0
> Property(S): PrimaryVolumeSpaceRemaining = 0
> MSI (s) (F4:D8) [06:05:08:336]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (F4:D8) [06:05:08:336]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (F4:D8) [06:05:08:336]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (F4:D8) [06:05:08:336]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (F4:D8) [06:05:08:336]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.

(Automated response - build 886.)

File: manifests/n/Nanoleaf/NanoleafDesktop/1.8.3/Nanoleaf.NanoleafDesktop.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+PackageIdentifier: Nanoleaf.NanoleafDesktop
+PackageVersion: 1.8.3
+InstallerType: nullsoft
+Installers:
+- Architecture: x86
+  InstallerUrl: https://my.nanoleaf.me/api/v2/desktopApp/latest/win
+  InstallerSha256: 27CCC41CEB3C6F18D9030B68750E3B0F407A0FB4C2A835A777E82B6C075A8C4B
+  Dependencies:
+    PackageDependencies:
+      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```
Comment (stephengillie) [inline manifests/n/Nanoleaf/NanoleafDesktop/1.8.3/Nanoleaf.NanoleafDesktop.installer.yaml:13]:
```suggestion
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @znarfm,

Our `x86` validations are performed on 32-bit OSes. This appears to be a 64-bit only application. 

image

**Comment by stephengillie** [comment]
Verify:
image
image

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
- https://github.com/microsoft/winget-pkgs/issues/142854

**Comment by Thulasi225** [comment]
Verified

**Pull request merged**